app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -138,7 +138,8 @@ sealed class Event {
             context.getString(R.string.pref_key_search_bookmarks),
             context.getString(R.string.pref_key_search_browsing_history),
             context.getString(R.string.pref_key_show_clipboard_suggestions),
-            context.getString(R.string.pref_key_show_search_shortcuts)
+            context.getString(R.string.pref_key_show_search_shortcuts),
+            R.string.pref_key_open_links_in_a_private_tab
```

Comment:
Do we need to wrap this in `context.getString`?

Reply:
Yup 🙃